File: 1716_Calculate_Money_in_Leetcode_Bank/solution.py
class Solution:
    def totalMoney(self, n: int) -> int:
        N_of_Weeks, N_of_Days = n // 7, n % 7
        Total_Amount_of_Weeks = (N_of_Weeks * (56 + (N_of_Weeks - 1) * 7)) // 2
        Total_Amount_of_Days = (N_of_Days * (2 * (N_of_Weeks + 1) + (N_of_Days - 1))) // 2
    
        return Total_Amount_of_Weeks + Total_Amount_of_Days

File: 1716_Calculate_Money_in_Leetcode_Bank/solution.c
int totalMoney(int n) {
    int N_of_Weeks = n / 7, N_of_Days = n % 7;
    int Total_Amount_of_Weeks = (N_of_Weeks * (56 + (N_of_Weeks - 1) * 7)) / 2;
    int Total_Amount_of_Days = (N_of_Days * (2 * (N_of_Weeks + 1) + (N_of_Days - 1))) / 2;
    
    return Total_Amount_of_Weeks + Total_Amount_of_Days;
}